rero_ils/modules/patron_transaction_events/jsonschemas/patron_transaction_events/patron_transaction_event-v0.0.1.json

```diff
@@ -42,9 +42,9 @@
     },
     "amount": {
       "title": "Amount",
+      "description": "The fee amount as cents",
       "type": "number",
```

Comment:
If we work with cents, why do not use integer as type ?
Another thought... why use cents if you check decimals with an extension?

Reply:
You're right ! Mistake.
I begin the work to play with cents, but I found a method to "nicely" work with decimals.